patroni/dcs/consul.py

```diff
@@ -232,6 +232,9 @@ def create_session(self):
                 logger.info('waiting on consul')
                 time.sleep(5)
 
+    def reload_config(self, config):
```

Comment:
You still have to call the [parent](https://github.com/zalando/patroni/blob/master/patroni/dcs/__init__.py#L592) method